pattern-library/js/edx-icons.js

```diff
@@ -1,9 +1,13 @@
-define(['afontgarde'],
-    function() {
-        'use strict';
+define(['afontgarde'], function() {
+    'use strict';
 
-        window.AFontGarde('edx-icons', {
-            // Check a few random icons to see if our font loads
-            glyphs: '\uE621\uE622\uE623'
-        });
+    // Old icon fonts, currently used in the platform.
+    window.AFontGarde('FontAwesome', {
```

Comment:
I guess I misunderstood you here @clrux. I thought that the Font Awesome icons addition would only be in the platform. I don't love the idea of having this codified in the Pattern Library too.